lib/opencensus/absinthe/plug.ex

```diff
@@ -28,8 +28,8 @@ defmodule Opencensus.Absinthe.Plug do
     """
     def traced_pipeline(config, pipeline_opts \\ []) do
       config
-      |> Absinthe.Plug.default_pipeline(pipeline_opts)
       |> Opencensus.Absinthe.add_phases()
+      |> Absinthe.Plug.default_pipeline(pipeline_opts)
```

Comment:
Did this work? If so, the Absinthe documentation might need some work. The documentation for [`Absinthe.Plug/default_pipeline/2`](https://hexdocs.pm/absinthe_plug/Absinthe.Plug.html#default_pipeline/2) says it takes a map and a keyword list and returns a pipeline, i.e. a list of phases…

Reply:
> Did this work? If so, the Absinthe documentation might need some work. The documentation for [`Absinthe.Plug/default_pipeline/2`](https://hexdocs.pm/absinthe_plug/Absinthe.Plug.html#default_pipeline/2) says it takes a map and a keyword list and returns a pipeline, i.e. a list of phases…

Probably not. As you say, this looks like I have lines 31 and 32 in the wrong order. 🤦‍♂ 

Nice catch!